src/main/java/com/labsynch/labseer/service/LotServiceImpl.java

```diff
@@ -608,4 +670,39 @@ public void deleteLot(Lot lot) {
 		lot.remove();
 	}
 
+	public enum LotOrphanLevel {
+		Parent, SaltForm, NONE
+	}
+
+	@Override
+	public LotOrphanLevel checkLotOrphanLevel(Lot lot) {
```

Comment:
New method  for reuse with delete lot to check if the salt form, parent, or none would be orphaned by the lot going away (reparented or deleted)